flowmachine/flowmachine/core/mixins/geodata_mixin.py

```diff
@@ -243,16 +244,32 @@ def _get_geojson(self, proj4):
         dict
 
         """
-        features = [
-            x[0] for x in self.connection.fetch(self._geo_json_query(crs=proj4))
-        ]
+        features = [x[0] for x in self.connection.fetch(self.geojson_query(crs=proj4))]
         js = {
             "properties": {"crs": proj4},
             "type": "FeatureCollection",
             "features": features,
         }
         return js
 
+    def get_proj4_string(self, crs=None):
+        """
+        Get the proj4 string corresponding to crs, or the default proj4 string if crs is None.
+
+        Parameters
+        ----------
+        crs : int or str
+            Optionally give an integer srid, or valid proj4 string
+        
+        Returns
+        -------
+        str
+            proj4 string
+        """
+        # TODO: Some refactoring would be beneficial here.
+        proj4_string = proj4string(self.connection, crs)
```

Comment:
Minor thing, but here we're creating a local variable with the same name as a pre-existing function, and then immediately return it. I suggest to simply change this to
```
return proj4string(self.connection, crs)
```